src/templates/utils/index.ts

```diff
@@ -146,6 +146,10 @@ export const validateTemplateURL = (url): string => {
   */
   const cleanUrl = url.trim().split('?')[0]
 
+  if (cleanUrl.includes(' ')) {
+    return `Your URL can't contain spaces`
```

Comment:
context nit: a string wrapped in \`backticks\` implies there will be string interpolation. Better to use double quotes here, which in our codebase are only used when there's a single quote in the string.